src/legacy/core_plugins/metrics/server/lib/vis_data/helpers/get_es_shard_timeout.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export default function getEsShardTimeout(req) {
```

Comment:
__note:__ @sulemanof please, please, please let's stop using `default` exports. It's really hard to grep if you'd like to find places where this function is used and we're trying to not use it for any new code.

On a related note, I'm working on the PR that will forbid direct access to `elasticsearch` config. We'll keep legacy escape hatch, but it will be asynchronous so please keep that in mind if you'd like to get something from ES config directly.

Reply:
I didn't event think the default export could mess the code =) 
Thanks for the note, will avoid it next time

Reply:
Np, thanks :+1: